File: @LV1/1.9/sumoftheothers/sumoftheothers.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	scan := bufio.NewScanner(os.Stdin)

	for scan.Scan() {
		s := strings.Split(scan.Text(), " ")

		sum := 0
		for _, n := range s {
			num, _ := strconv.Atoi(n)
			sum += num
		}

		fmt.Println(sum / 2)
	}
}

File: @LV1/1.9/sumoftheothers/sumoftheothers.cpp
#include <cstdio>
#include <cstdlib>
#include <cstring>

int main() {
    char s[1000];

    while (fgets(s, 1000, stdin)) {
        s[strlen(s) - 1] = '\0';
        int sum = 0;
        char *p = strtok(s, " ");

        while (p != NULL) {
            sum += atoi(p);
            p = strtok(NULL, " ");
        }
        printf("%d\n", sum / 2);
    }
}
